CITATION.cff

```diff
@@ -0,0 +1,76 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+---
+cff-version: 1.2.0
+title: pycytominer-transform
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: David
+    family-names: Bunten
+    orcid: 'https://orcid.org/0000-0001-6041-3665'
+  - given-names: Gregory
+    family-names: Way
+    orcid: 'https://orcid.org/0000-0002-0503-9348'
+  - given-names: Faisal
+    family-names: Alquaddoomi
+    orcid: 'https://orcid.org/0000-0003-4297-8747'
+repository-code: 'https://github.com/cytomining/pycytominer-transform'
+url: 'https://cytomining.github.io/pycytominer-transform/'
+abstract: >-
+  Transform data for processing image-based profiling
+  readouts with Pycytominer.
+keywords:
+  - microscopy
+  - profiling
+  - data
+  - data-engineering
+  - pycytominer
+  - way-lab
+license: BSD-3-Clause
+references:
+  - authors:
+      - name: "cytominer-database Development Team"
```

Comment:
Thanks for catching this! Adding this into a forthcoming commit.

Reply:
@gwaybio - circling back to this I noticed that the suggestion might not have included the extra space removal you mentioned (I thought it may have been after the final quote). It may also be that the Github suggestion is doing something unusual, not certain. Could I ask for more clarification here? Thanks!

Reply:
yeah, I think that it is behaving strangely! It looks good to me as it is now

Reply:
Thanks! I'll mark this as resolved.